src/pages/AnimeOnsen/main.ts

```diff
@@ -74,15 +115,22 @@ export const AnimeOnsen: pageInterface = {
       // handlePage()
       page.handlePage();
 
-      // go-home + episode-next
-      j.$('div#ao-episode-buttons-wrapper').on('click', () => {
+      // handle going back
+      j.$('div.ao-player-metadata-navigation button').on('click', () => {
+        setTimeout(() => {
+          page.handleList();
```

Comment:
handleList is used to update the episode List if for example the page has a pagination. What exactly was the idea behind this. Did I miss something?

Reply:
in the UI, this is the burger button to go back to the details page, it used to be the previous episode button on an old version of the site.

Reply:
It is not needed if there is a page load in between. It is just for the case you have a episode pagination and you switch the pagination page. Means in case the overview page stays but the episode list changes afterwards which is here not the case